pkg/controller/pod.go

```diff
@@ -307,12 +318,38 @@ func (pnc *PodNetworksController) removeNetworks(dynamicAttachmentRequest *Dynam
 		}
 		klog.Infof("response: %v", *response)
 
+		newIfaceStatus, err := annotations.DeleteDynamicIfaceFromStatus(
+			pod,
+			namespacedName(netToRemove.Namespace, netToRemove.Name),
```

Comment:
Let's just pass `netToRemove` and do the namespacedName inside it.

Reply:
Why ? It doesn't need anything else.

Reply:
to make "AddDy" and "DeleteDyn" more symmetric is weird that Add use a proper struct while Delete use a string.